### PR TITLE
[ALLUXIO-2897] Changed "in-memory" to "in-Alluxio" wherever applicable

### PR DESCRIPTION
**Pull request by prasadns14**
https://alluxio.atlassian.net/browse/ALLUXIO-2897

Changed "in-memory" to "in-Alluxio" wherever applicable. 

There are many test cases which are failing because of these changes. Please verify the changes, I will fix the failing test cases once the changes are approved.

### CODE REVIEW

**Comment by prasadns14** [comment]
Made changes as per the review. And also fixed the failing test cases, except the below two-
1) alluxio.master.file.FileSystemMasterTest.isFullyInMemory
2) alluxio.worker.block.meta.TierPromoteIntegrationTest.promoteBlock

Need help in understanding the test case

File: core/common/src/thrift/file_system_master.thrift [hunk 1]
```diff
@@ -103,7 +103,7 @@ struct FileInfo {
   11: bool cacheable
   12: bool persisted
   13: list<i64> blockIds
-  15: i32 inMemoryPercentage
+  15: i32 inMemoryPercentage // deprecated (replaced by inAlluxioStorage)
```
Comment (aaudiber) [inline core/common/src/thrift/file_system_master.thrift]: replaced by inAlluxioPercentage

File: docs/en/Running-Alluxio-Locally.md [hunk 2]
```diff
@@ -69,7 +69,8 @@ You can stop Alluxio any time by running:
 
 ## Why is sudo privilege needed to start Alluxio on Linux?
 
-By default, Alluxio filesystem uses [RAMFS](https://www.kernel.org/doc/Documentation/filesystems/ramfs-rootfs-initramfs.txt) as its in-memory data storage. It turns out on MacOS, it is fine for Alluxio to mount a RAMFS without being a super user. However, on Linux, it requires sudo privileges to perform "mount" (and the followed "umount", "mkdir" and "chmod" operations). 
+By default, Alluxio filesystem uses [RAMFS](https://www.kernel
```
Comment (aaudiber) [inline docs/en/Running-Alluxio-Locally.md]: I think we need to remove the linebreak for the link to work

**Comment by prasadns14** [comment]
Thanks for the review @aaudiber!
I have resolved the conflict.

**Pull request merged**